src/tctrack/tempest_extremes/tempest_extremes.py

```diff
@@ -842,7 +842,7 @@ def _parse_gfdl_line_to_point(
         Returns
         -------
         tuple
-            A tuple of integer year, day, month, hour and dict of variables
+            A tuple of an integer list for the time and a dict of variables
```

Comment:
I wonder if it's still worth being explicit about units and order here `[year, day, month, hour]`?